### PR TITLE
PR for landing page

### PR DESCRIPTION
**Pull request by manismk**
- Added landing page for the Notes app
- Added router

image

### CODE REVIEW

**Review by mohammadkhizerkhan — APPROVED** [review]
everything looks good ,

File: src/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { Home } from "./home/Home";
```
Comment (SK02K1) [inline src/pages/index.js]: - No need to import ```Home``` for exporting.
- We can directly export it as

```suggestion
export { Home } from "./home/Home";
```

File: src/assets/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import heroImg from "../assets/hero.svg";
```
Comment (SK02K1) [inline src/assets/index.js:1]: - Liked the way you used this approach for assets.

**Review by SK02K1 — APPROVED** [review]
- The code is clean and well structured.
- Just one suggestion that you can look into before merging.

**Pull request merged**